src/components/Menu/styles.module.css

```diff
@@ -18,3 +19,7 @@
   padding-left: 15px;
   margin-top: 15px;
 }
+
+.nav a {
```

Comment:
Maybe I'm missing something, but this is probably not needed unless you have a reason to do so.